src/features/common/api/api.ts

```diff
@@ -1,17 +1,13 @@
 import axios from 'axios';
+import {ResponseDataType} from "../types/types";
 
 const instance = axios.create({
     baseURL: 'https://reqres.in/api'
 })
 
-export const getAllUsers = async () => {
-    const response = await  fetch('https://reqres.in/api/users');
-
-    return response.json()
-};
-// export const getAllUsers = async () => {
-//     const response = await instance.get<any>('/users');
-//
-//     return await response.data.json()
-// };
-
+export const reactTestProjectAPI = {
+    getAllUsers: async () => {
```

Comment:
I think we could try improving the naming. It seems like we get only users from page 1, so I would probavly suggest having the name `getUsersOnPage` and take a `page` as a function parameter, something like this:

```js
const getUsersOnPage = (page) => (await instance.get<ResponseDataType>(`/users?page=${page}`)).data
```

Reply:
By the way, what happens if we have an error while fetching the users data? It seems like we should handle it somehow.